### PR TITLE
Add schema BreadcrumbList

### LINKED ISSUE #2813: Schema.org - Add support for breadcrumbs
Add support for breadcrumbs as per Google doc here - https://developers.google.com/search/docs/data-types/breadcrumb

### PR DESCRIPTION
**Pull request by EkoJR**
Resolves #2813

## Proposed changes

Adds breadcrumbs to the schema structured data.

## Types of changes

What types of changes does your code introduce?

- New feature (non-breaking change which adds functionality)
- Improves existing functionality

## Checklist

- [x] I've selected the appropriate branch (ie x.y rather than master).
- [x] I've created an appropriate title, descriptive of what the PR does.
- [ ] Travis-ci runs with no errors.
- [ ] I have added tests that prove my fix is effective/my feature works or have created an issue for it (if appropriate).
- [ ] I have added necessary documentation, or have created an issue for docs (if appropriate).

## Testing instructions
1. Navigate frontend and check ld+/json data in <head>
    a. Home/Main
    b. Post
    c. Child Page
    d. Taxonomy Term archive.
    e. Date archive.
    f. Author page.
    g. Search page.

## Further comments

If this is a relatively large or complex change, kick off the discussion by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

File: inc/schema/aioseop-context.php [hunk 1]
```diff
@@ -398,26 +422,44 @@ public static function get_object( $type, $key, $args = array() ) {
...
+			case 'var_date_day':
 				$object = array(
 					'context_type' => $type,
 					'context_key'  => $key,
 				);
 				break;
+
 			case 'WP_Site':
+				/*
+				 * PHP 5.2 conflict. Could merge WP_Site & WP_Post together after WP 5.1 becomes the required version.
```
Comment (benjaminprojas) [inline inc/schema/aioseop-context.php]: What's the actual conflict that happens here?

**Comment by benjaminprojas** [comment]
@EkoJR is there supposed to be a `BreadcrumbList` on the home page? I'm not seeing one there at all, which is okay if its not meant to show. I just see this:

```JSON
...
"breadcrumb": {
	"@id": "https://aioseo.local#breadcrumblist"
},
...
```

File: inc/schema/aioseop-context.php (hunk 1 above)
Comment (EkoJR) [reply inc/schema/aioseop-context.php]: It was an issue with dynamically accessing a method.

`Parse error: syntax error, unexpected T_PAAMAYIM_NEKUDOTAYIM`

**Comment by EkoJR** [comment]
@wpsmort @benjaminprojas The error should now be fixed on the Homepage.

**Comment by EkoJR** [comment]
@benjaminprojas Added a fix for handling `$wp_query->is_posts_page` instance/condition.

**Comment by wpsmort** [comment]
This is tested and is good.

**Pull request merged**